dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -299,6 +303,42 @@ internal class RumViewScope(
         sendViewUpdate(event, writer)
     }
 
+    private fun onApplicationStarted(
+        event: RumRawEvent.ApplicationStarted,
+        writer: Writer<RumEvent>
+    ) {
+        val now = event.eventTime.nanoTime
+        val startupTime = Datadog.startupTimeNs
+        val context = getRumContext()
+        val actionEvent = ActionEvent(
+            date = eventTimestamp,
+            action = ActionEvent.Action(
+                type = ActionEvent.Type1.APPLICATION_START,
+                id = UUID.randomUUID().toString(),
+                loadingTime = max(now - startupTime, 1L)
+            ),
+            view = ActionEvent.View(
+                id = context.viewId.orEmpty(),
+                url = context.viewUrl.orEmpty()
+            ),
+            application = ActionEvent.Application(context.applicationId),
+            session = ActionEvent.Session(
+                id = context.sessionId,
+                type = ActionEvent.Type.USER
+            ),
+            dd = ActionEvent.Dd()
+        )
+        val rumEvent = RumEvent(
+            event = actionEvent,
+            attributes = emptyMap(),
+            userInfo = RumFeature.userInfoProvider.getUserInfo()
+        )
+        writer.write(rumEvent)
+
+        actionCount++
```

Comment:
@xgouchet Is this the only reason why we move `application_start` event to View Scope? To increment view's action count?

Reply:
Nope, it was to use the actual view context too

Reply:
Also I updated the timing to make this event start at the same time as the view, otherwise it doesn't appear in the explorer

Reply:
Oh, thanks 👌. I will do similar on iOS 👍.